vagrant/src/main/vagrant/files/install_brooklyn.sh

```diff
@@ -78,12 +78,12 @@ tar zxf ${TMP_ARCHIVE_NAME}
 echo "Creating Brooklyn dirs and symlinks"
 ln -s ${BROOKLYN_DIR} apache-brooklyn
 sudo mkdir -p /var/log/brooklyn
-sudo chown -R vagrant:vagrant /var/log/brooklyn
-mkdir -p /home/vagrant/.brooklyn
+sudo chown -R ubuntu:ubuntu /var/log/brooklyn
+mkdir -p /home/ubuntu/.brooklyn
```

Comment:
Why move to ubuntu? The benefit of the vagrant user is that it's automounted on the host OS so you can get at config / transfer files. That said if it's not in the getting started steps then it's probably fine to switch.